### ISSUE TITLE
Port for Weechat

### ISSUE THREAD

**Issue by hahuang65** [arcticicestudio/nord#21]
I'd actually like to use the nord colors in Weechat. Problem is, I'm not sure how to translate the terminal colors to Weechat.

if you're not familiar with Weechat, it's an IRC client that runs in your terminal. So I've got my terminal set up with Nord (I use both iTerm2 and urxvt), but my Weechat is configured to use more colors than Nord defines:
```
/set weechat.color.chat_nick_colors "cyan,magenta,green,brown,default,lightcyan,lightmagenta,blue,31,35,38,40,49,63,70,80,92,99,112,126,130,138,142,148,160,162,167,169,174,176,178,184,186,210,212,215,247"
```

The documentation for Weechat's colors is here:
http://weechat.org/files/doc/stable/weechat_user.en.html#command_line_colors

How might I reconcile the 2 so that I only get Nord defined colors in Weechat?

**Comment by arcticicestudio** [comment]
I'm a Weechat user too 😉
I haven't got the time to create a Nord theme for it because I'm busy with my real life coding job and try to maintain and create other Nord projects takes a lot of time.

I've added it to port project backlog #18 in the **CLI Applications** section so you can track the progress. I'll @-mention you as son as I create the backlog issue and start working on it.

**Comment by hahuang65** [comment]
Yup, just saw that. I might take some free time and look into making it work for weechat if you'll accept PRs :)

Keep up the good work... This theme fricking rocks!
